### PR TITLE
added options to include/exclude lines

### PR DESCRIPTION
**Pull request by jeremyj**

### CODE REVIEW

**Comment by idohalevi** [comment]
@jeremyj Thank you!
Any reason why exclude/include should be together in the same config file? why can't we have if/else relation between them?

**Comment by jeremyj** [comment]
Hey @idohalevi no problem. It's just a quick hack for something I needed.
Being pretty much a novice both to logz.io and to filebeat, I'm not sure I undestand your quesitons. Can you elaborate?

**Comment by idohalevi** [comment]
@jeremyj Sure. I see that in lines 172-176 you have two if. My understanding is that you either want to include or exclude but not both so I was wondering why it's not an if/else statement

**Comment by jeremyj** [comment]
Hi @idohalevi 
The Filebeat documentation says that we can have `include_lines` and `exclude_lines` together. If both are defined, `include_lines` is executed first.

File: filebeat-yml-script.py [hunk 1]
```diff
@@ -145,5 +169,10 @@ def _include_containers():
 else:
     _exclude_containers()
 
+if "excludeLines" in os.environ:
+    _exclude_lines()
+
+if "includeLines" in os.environ:
+    _include_lines()
```
Comment (idohalevi) [inline filebeat-yml-script.py]: @jeremyj According to your comment we should change the order of exclude/include here

**Comment by idohalevi** [comment]
> Hi @idohalevi
> The Filebeat documentation says that we can have `include_lines` and `exclude_lines` together. If both are defined, `include_lines` is executed first.

Thank you! Just switch the order of include/exclude and I'll make sure to merge the PR

**Comment by jeremyj** [comment]
From the Filebeat documentation: "The order in which the two options are defined doesn’t matter. The include_lines option will always be executed before the exclude_lines option, even if exclude_lines appears before include_lines in the config file."

**Pull request merged**